cmd/up/up.go

```diff
@@ -327,9 +327,6 @@ func (up *upContext) getApp(ctx context.Context) (apps.App, bool, error) {
 		return apps.NewDeploymentApp(apps.GetDeploymentSandbox(up.Dev)), true, nil
 	}
 	if err == nil {
-		if app.ObjectMeta().Annotations[model.OktetoAutoCreateAnnotation] != model.OktetoUpCmd {
-			up.isSwap = true
```

Comment:
Just curiosity, what was the meaning of this flag?

Reply:
it was related to `autocreate`, before existing the `autocreate` field